### DISCUSSION TITLE
Welcome to APK-Parser Discussions!

### DISCUSSION THREAD

**Discussion by MALTF** [MALTF/APK-Parser#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RickieSayHi** [comment]
woops, 1st floor

**Reply by MALTF** [reply]
I got the badge without doing any of the things mentioned above, I simply made commits on another branch to add a feature and merged the branch into main. You can see my actual PR https://github.com/ajmeese7/8bit/pull/1, as evidenced by the badge information on my user profile.

Accept if this helps, I'm looking forwards to getting the discussion award too 😄

**Reply by RickieSayHi** [reply]
that's it!

**Comment by bingolovess** [comment]
Hope to create a tutorial.

**Reply by MALTF** [reply]
look this https://github.com/Schweinepriester/github-profile-achievements/discussions/6
